### DISCUSSION TITLE
Master or main?

### DISCUSSION THREAD

**Discussion by Protiguous** [AdamWhiteHat/BigDecimal#32] [Discussions]
@AdamWhiteHat, @PixelatedLagg

I would like the prime branch to be changed to “main” instead of the current “master”.

**Comment by Protiguous** [comment]
@AdamWhiteHat Any idea on changing the name of the main branch?

**Comment by AdamWhiteHat** [comment]
There. As per your request, I made the default branch 'main'.

**Reply by Protiguous** [reply]
Thank you.
